### ISSUE TITLE
Crash dump after changing settings when sleep mode is set to true

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4451]
**Reported by norrumar on 2014-09-11 12:38**
Hi, I don't know exactly when NVDA has crashed. I couldn't restart it and I have needed to restart the computer.
I have stored an app module setting sleep mode variable to true, cause in an application the cursor doesn't work properly otherwise. I wanted to test if this could be fixed changing settings of the review cursor, for instance: don't follow the system cursor.
Then NVDA has failed and a crytical error has been logged in nvda-old.log.
Thanks

**Comment by bhavyashah** [comment]
@nvdaes Does the reported crash still occur? If so, could you please test this with all add-ons disabled and then share your findings?

**Comment by nvdaes** [comment]
Hi @bhavyashah, I don't have this problem now. You can close this. Cheers

**Comment by ehollig** [comment]
Closing, as requested in https://github.com/nvaccess/nvda/issues/4451#issuecomment-325318937
